pyproject.toml

```diff
@@ -32,7 +32,7 @@ semver = "^2.10.2"
 gitpython = "^3.1.35"
 requests = "2.27.1"
 locust = "^1.2"
-pytest-bdd = "^4.0.1"
+pytest-bdd = "^4.1.0"
```

Comment:
Required a bump up to allow for `when` and `then` steps to define `target_fixtures` - see https://pytest-bdd.readthedocs.io/en/stable/#id38